programs/zstdcli.c

```diff
@@ -672,7 +673,23 @@ int main(int argCount, const char* argv[])
         }
     }
 #endif
-
+    if(operation==zom_list){
+        if(filenameIdx==0){
+            DISPLAY("No files given\n");
+            CLEAN_RETURN(0);
+        }
+        DISPLAY("===========================================\n");
```

Comment:
By default, `DISPLAY` writes into `stderr`.
That's fine for warning and error messages.
But when it's an intentional product, it's better to write into `stdout`.

The way to achieve this redirection within `zstdcli`
is to set `g_displayOut = stdout;` before `DISPLAY`.